doc/Getting_Started.md

```diff
@@ -172,7 +172,8 @@ This will build the app and generate a binary in Tock Binary Format
 `userland/examples/blink/build/cortex-m4/cortex-m4.bin`.
 
 Alternatively, apps can be built and automatically uploaded from the
-Tock root directory:
+Tock root directory: (this is currently only available for boards that support
+`program` i.e. programming via serial/USB):
```

Comment:
This isn't right any more. There's no `program` target in the root directory for any boards now